### ISSUE TITLE
Question: pagination not works for me

### ISSUE THREAD

**Issue by KhodeN** [biosustain/potion#66]
I have this resource:

```python
class Goods(ModelResource):
    class Meta:
        name = 'goods'
        model = dal.Good
        include_id = True

    class Schema:
        category = fields.ToOne('good_categories')
```

If I send GET request to http://localhost:5000/goods, response doesn't contain `Links` and `X-Total-Count`headers.

in `instances.py` there is some code like this:
```python
class PaginationMixin(object):
  def format_response(self, data):
        if not isinstance(data, self._pagination_types): 
            return self.format(data) # always RETURN THIS, break rest of the method
        .... 
```

**Comment by lyschoening** [comment]
Are you using Python 3 and importing your modules using `flask.ext.*`?

**Comment by KhodeN** [comment]
I know this trouble of pycharm)

No, my import code is (python 3)

```
from flask_potion import ModelResource, fields
```

**Comment by lyschoening** [comment]
What manager are you using? SQLAlchemy? If so, are you importing `flask_sqlalchemy`?

**Comment by KhodeN** [comment]
```

**Comment by lyschoening** [comment]
Ok, it looks like the pagination for the PeeweeManager is not implemented correctly. Peewee's built-in pagination doesn't return a count, so we'll need to write something custom. I can look at it tomorrow.

**Comment by KhodeN** [comment]
Thanks, I will be waiting...

**Comment by lyschoening** [comment]
Fixed in 0.12.1
